adapters/applogy/applogy.go

```diff
@@ -70,7 +70,7 @@ func (a *ApplogyAdapter) MakeRequests(request *openrtb.BidRequest, _ *adapters.E
 			errs = append(errs, errors.New("Applogy token required"))
 			continue
 		}
-		request.Imp = impressions[i : i+1]
+		request.Imp = []openrtb.Imp{impression}
```

Comment:
I think this the best approach for this adapter. Based on the `request.Imp = impressions` line later in this adapter, it seems their intent is to preserve the bid request. Any other approach would not accomplish that goal.